src/View/Components/Badge.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace WireUi\View\Components;
+
+class Badge extends BaseBadge
+{
+    public function getDefaults()
```

Comment:
This method is complex, split into two methods to reduce complexity
You can use the `classes` function to apply conditional CSS classes